deployment/build-s3-dist.sh

```diff
@@ -22,20 +24,26 @@ if [ -z "$1" ] || [ -z "$2" ] || [ -z "$3" ]; then
     exit 1
 fi
 
-export DIST_VERSION=$3
 export DIST_OUTPUT_BUCKET=$1
-export SOLUTION_ID=SO0030
 export SOLUTION_NAME=$2
-export SOLUTION_TRADEMARKEDNAME=$2
+export VERSION=$3
 
 # Get reference for all important folders
-template_dir="$PWD"
-staging_dist_dir="$template_dir/staging"
-global_dist_dir="$template_dir/global-s3-assets"
-regional_dist_dir="$template_dir/regional-s3-assets"
-lambda_source_dir="$template_dir/../source/app"
-cli_source_dir="$template_dir/../source/cli"
-cdk_source_dir="$template_dir/../source/infrastructure"
+project_root="$PWD/.."
+deployment_dir="$PWD"
```

Comment:
No need to change this now, but in general I prefer to base paths relative to the file rather than the calling location. This can be done with something like: `local root_dir=$(dirname "$(cd -P -- "$(dirname "$0")" && pwd -P)")`